Documentation/Authors.md

```diff
@@ -1,133 +0,0 @@
-# Authors
```

Comment:
Can we move this file to the repo root? I'm not sure it was migrated to DMC and I'd like to keep this list around!

Reply:
Sure!